lib/collections/schemas/shops.js

```diff
@@ -88,6 +89,52 @@ export const Currency = new SimpleSchema({
 
 registerSchema("Currency", Currency);
 
+/**
+ * @name Currency
+ * @memberof schemas
+ * @type {SimpleSchema}
+ * @property {Number} weight default value: 8lb
+ * @property {Number} height default value: 6in
+ * @property {Number} length default value: 11.25in
+ * @property {Number} width default value: 8.75in
+ */
+export const ParcelSize = new SimpleSchema({
+  weight: {
+    type: Number,
+    defaultValue() {
+      const baseUOM = this.siblingField("baseUOM");
+      return convertWeight("lb", baseUOM, 8);
+    },
```

Comment:
`defaultValue` as a function doesn't work. You could switch to `autoValue`, but the logic will have to be more complex than this. I think it's easier to just add these defaults wherever you're inserting/updating and remove these defaults from the schema.